felix/bpf-gpl/parsing.h

```diff
@@ -10,6 +10,66 @@
 #define PARSING_ALLOW_WITHOUT_ENFORCING_POLICY 2
 #define PARSING_ERROR -1
 
+#define MAX_EXTENSIONS 2
+
+static CALI_BPF_INLINE int parse_packet_ipv6(struct cali_tc_ctx *ctx, int iterations) {
+	__u8 next_header = ipv6_hdr(ctx)->nexthdr;
+
+	for (int i = 0; i < iterations; i++) {
+		if (skb_refresh_validate_ptrs(ctx, UDP_SIZE)) {
```

Comment:
we should know the value for each iteration, right? I can see where the verifier may have troubles. We should imo really se the get_data to get next_header

Reply:
Address this in another PR.